static/js/script.js

```diff
@@ -27,9 +27,10 @@ if (flashes) {
 /* display profile social network field on click */
 const socialIcon = document.querySelector('.social-networks li')
 if (socialIcon) {
-    Array.from(document.querySelector('.social-networks li')).forEach((li) => {
-            socialIcon.addEventListener('click', (event) => {
-                event.preventDefault()
+    Array.from(socialIcon).forEach((li) => {
+        console.log('oink')
+        li.addEventListener('click', (event) => {
```

Comment:
If it's clickable, it should be a `button` or a `a`, else this is just fully breaking any accessibility for sure.

Reply:
oui